app/javascript/packs/articlePage.jsx

```diff
@@ -6,6 +6,9 @@ const snackZone = document.getElementById('snack-zone');
 
 render(<Snackbar lifespan="3" />, snackZone, snackZone.firstElementChild);
 
+// eslint-disable-next-line no-restricted-globals
+top.addSnackbarItem = addSnackbarItem;
```

Comment:
what is this hack for?

Reply:
The actions panel is loaded in an iframe, so to reference the proper `snackbar` we have to assign it to `top` so it's available globally. :(

Won't be problem once we make the actions panel via Preact.